src/atomate2/vasp/flows/phonons.py

```diff
@@ -0,0 +1,318 @@
+        if self.kpath_scheme not in [
+            "seekpath",
+            "hinuma",
+            "setyawan_curtarolo",
+            "latimer_munro",
+            "all",
+        ]:
+            raise ValueError("kpath scheme is not implemented")
+
+        jobs = []
+
+        if self.use_symmetrized_structure == "primitive":
+            # These structures are compatible with many
+            # of the kpath algorithms that are used for Materials Project
+            prim_job = structure_to_primitive(structure, self.symprec)
+            jobs.append(prim_job)
+            structure = prim_job.output
+        elif self.use_symmetrized_structure == "conventional":
+            # it could be beneficial to use conventional
+            # standard structures to arrive faster at supercells with right
+            # angels
+            conv_job = structure_to_conventional(structure, self.symprec)
+            jobs.append(conv_job)
+            structure = conv_job.output
+
+        # if supercell_matrix is None, supercell size will be determined
+        if supercell_matrix is None:
+            supercell_job = get_supercell_size(
+                structure,
+                self.min_length,
+                self.prefer_90_degrees,
+                **self.get_supercell_size_kwargs,
+            )
+            jobs.append(supercell_job)
+            supercell_matrix = supercell_job.output
+
+        if self.bulk_relax_maker is not None:
+            # optionally relax the structure
+            bulk = self.bulk_relax_maker.make(structure, prev_vasp_dir=prev_vasp_dir)
+            jobs.append(bulk)
+            structure = bulk.output.structure
+
+        # get a phonon object from phonopy
+        displacements = generate_phonon_displacements(
+            structure=structure,
+            supercell_matrix=supercell_matrix,
+            displacement=self.displacement,
+            sym_reduce=self.sym_reduce,
+            symprec=self.symprec,
+            use_symmetrized_structure=self.use_symmetrized_structure,
+            kpath_scheme=self.kpath_scheme,
+            code=self.code,
+        )
+        jobs.append(displacements)
+
+        # perform the phonon displacement calculations
+        vasp_displacement_calcs = run_phonon_displacements(
+            displacements=displacements.output,
+            structure=structure,
+            supercell_matrix=supercell_matrix,
+            phonon_maker=self.phonon_displacement_maker,
+        )
+        jobs.append(vasp_displacement_calcs)
+
+        # Computation of static energy
+        if self.static_energy_maker is not None and total_dft_energy is None:
+            static_job = self.static_energy_maker.make(structure=structure)
+            jobs.append(static_job)
+            total_energy = static_job.output.output.energy
+            static_run_job_dir = static_job.output.dir_name
+            static_run_uuid = static_job.output.uuid
+        else:
+            total_energy = total_dft_energy
```

Comment:
And then you can remove this line

Reply:
Thanks. Will also change all other side effects of this change. I should probably run some of the automatic systems already now to spot such things. I wanted to wait until the unit tests are implemented as I always suspect potential problems.